assets/web-ui/index.html

```diff
@@ -32,10 +32,8 @@ <h1>SPX Control Panel</h1>
 
         <div id="reports"></div>
 
-        <script src="js/jquery-3.2.1.min.js"
-                integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="></script>
-
         <!-- Required workaround for Firefox to fix a "no credentials" issue -->
+        <script type="module" crossorigin src="?SPX_UI_URI=/js/jquery-3.2.1.min.js"></script>
```

Comment:
Is it possible to keep the integrity check `integrity="..."` here (same point for report.html) ?

Reply:
And `type="module"` is actually not necessary here, I was wrong yesterday.

Reply:
Something like that in the end ?
```suggestion
        <script crossorigin src="?SPX_UI_URI=/js/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="></script>
```